palantir-java-format-benchmarks/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'distribution'
+apply plugin: 'me.champeau.jmh'
+
+dependencies {
+    jmhAnnotationProcessor 'org.openjdk.jmh:jmh-generator-annprocess'
+    compileOnly 'org.openjdk.jmh:jmh-generator-annprocess'
+
+    jmhImplementation 'org.assertj:assertj-core'
+    jmhImplementation 'org.junit.jupiter:junit-jupiter'
+    jmhImplementation 'org.openjdk.jmh:jmh-core'
+    jmhImplementation 'org.openjdk.jmh:jmh-generator-annprocess'
+}
+
+tasks.named('processJmhResources', ProcessResources.class) {
+    from(project(':palantir-java-format').tasks.named('fatJar')) {
+        rename { 'palantir-java-format-all.jar' }
+    }
+    from(project(':palantir-java-format').tasks.named('nativeCompile')) {
+        rename { 'palantir-java-format'}
+    }
```

Comment:
We should make an outgoing artifact/configuration for nativeImage. I think you just need to do something like (wildly untested):

```gradle
configurations {
    nativeImage {
        outgoing.artifact(tasks.named('nativeImage'))
    }
}
```

Reply:
Then you make a configuration in this project and add a dep like `consumingProjectConfiguration project(name: ':palantir-java-format', configuration: 'nativeImage')`, then use that